### PR TITLE
Fc test files

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: lib/narwhal.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   this.rotateLeft = false;
   this.rotateRight = false;
   this.flipperKick = false;
   this.context = context;
   this.canvas = canvas;
+  this.tuskX = 414;
+  this.tuskY = 180;
+  this.tuskWidth = 2;
+  this.tuskHeight = 90;
 }
```
Comment (Obleo33) [inline lib/narwhal.js:17]: Added code for a tusk

File: lib/narwhal.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
 function narwhal(context,canvas) {
   this.x = 400;
   this.y = 250;
-  this.width = 40;
-  this.height = 30;
+  this.width = 30;
+  this.height = 40;
   this.speed = 2;
```
Comment (Obleo33) [inline lib/narwhal.js:6]: flipped hight and width so the narwhal starts pointing up

File: lib/narwhal.js [hunk 3]
```diff
@@ -18,6 +22,7 @@ narwhal.prototype.popNarwhal = function(){
   this.context.rotate(this.angle);
   this.context.translate(-this.x,-(this.y + this.height/2));
   this.context.fillRect(this.x,this.y,this.width,this.height)
+  this.context.fillRect(this.tuskX,this.tuskY,this.tuskWidth,this.tuskHeight)
```
Comment (Obleo33) [inline lib/narwhal.js:25]: Draw tusk to the page

File: lib/narwhal.js [hunk 4]
```diff
@@ -29,8 +34,10 @@ narwhal.prototype.updateNarwhal = function() {
       this.angle += Math.PI / 180;
     }
     if (this.flipperKick) {
-      this.x += 2 * (Math.sin(this.angle));
-      this.y += -2 * (Math.cos(this.angle));
+      this.x += 3 * (Math.sin(this.angle));
+      this.tusk += 3 * (Math.sin(this.angle));
+      this.y += -3 * (Math.cos(this.angle));
+      this.tuskY += 3 * (Math.cos(this.angle));
```
Comment (Obleo33) [inline lib/narwhal.js:40]: added code to move the tusk with the body. This isn't working

File: lib/narwhal.js (hunk 2 above)
Comment (juliankyer) [reply lib/narwhal.js:6]: Your hair is looking ok this morning. Also good call on re-orienting narwhal.

File: lib/narwhal.js (hunk 4 above)
Comment (juliankyer) [reply lib/narwhal.js:40]: Cool, we can get that going today.

**Pull request merged**